### PR TITLE
feat: hide <Html /> when occluded by an object 

### PR DESCRIPTION
**Pull request by drcmda**
### Checklist

- [ ] Documentation updated
- [ ] Storybook entry added
- [ ] Ready to be merged

### CODE REVIEW

File: src/web/Html.tsx [hunk 1]
```diff
@@ -89,6 +92,43 @@ export const Html = React.forwardRef(
...
+
+          const v1 = new THREE.Vector2(...position)
+          const v2 = new THREE.Vector2(...targetPosition)
+          
+          setOpacity(v1.distanceTo(v2)  / 100)
+        } else {
+          setOpacity(1)
+        }
+        
+        // const dist = v1.distanceTo(p.point)
```
Comment (gsimone) [inline src/web/Html.tsx:124]: Remove this in favore of a simple on/off

File: src/web/Html.tsx [hunk 2]
```diff
@@ -114,11 +154,12 @@ export const Html = React.forwardRef(
...
           left: -size.width / 2,
           width: size.width,
-          height: size.height,
+          height: size.height
         }),
+        opacity,
         ...style,
       }),
-      [style, center, fullscreen, size]
+      [style, center, fullscreen, size, opacity]
```
Comment (gsimone) [inline src/web/Html.tsx:162]: remove opacity, go display

File: .storybook/stories/HTML.stories.tsx [hunk 3]
```diff
@@ -82,3 +82,28 @@ export const HTMLCalculatePosition = () => (
   <HTMLScene className="html-story-label" calculatePosition={overrideCalculatePosition} />
 )
 HTMLCalculatePosition.storyName = 'Custom Calculate Position'
+
+
+function HTML2Scene(htmlProps: HtmlProps) {
```
Comment (gsimone) [inline .storybook/stories/HTML.stories.tsx:87]: Rename

**Comment by gsimone** [comment]
Note to self: make sure this plays well with the existing changes in Html

**Review by joshuaellis — COMMENTED** [review]
You've deleted poor suzanne

**Comment by gsimone** [comment]
> You've deleted poor suzanne

don't worry, I just quickly pushed to show @drcmda, I'll clean up the PR

File: src/web/Html.tsx (hunk 1 above)
Comment (drcmda) [reply src/web/Html.tsx:124]: "in favore" 🤌

**Comment by gsimone** [comment]
Closing in favor of #341

**Pull request closed without merging**